Doxyfile.in

```diff
@@ -776,11 +776,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = @top_srcdir@/doc/coding_standard_c.md \
-                         @top_srcdir@/doc/logging.md \
-                         @top_srcdir@/doc/doxygen.dox \
-                         @top_srcdir@/README.md \
-                         @top_srcdir@/include/tss2/tss2_esys.h \
+INPUT                  = @top_srcdir@/include/tss2/tss2_esys.h \
                          @top_srcdir@/src/tss2-esys \
```

Comment:
Why did you remove all the doc and dox files ?

Reply:
Right, that was probably too much. Updated